FUNCTIONS.md

```diff
@@ -0,0 +1,21 @@
+# Presto Function Guidelines
+
+Presto includes a large library of built-in, core SQL functions such as AVG and CONCAT that are available in every deployment.
+
+Presto also includes extension functions that can optionally be enabled in particular deployments.
+
+Finally, Presto allows users to write user defined functions (UDFs) in either Java or SQL that are not stored in this Github repository.
+
+This document lays out some guidelines for which category is right for which functions. 
+
+Functions defined by the ANSI SQL spec should be written as core functions in this repository.
```

Comment:
Suggest adding a link to the ANSI SQL spec.

Reply:
Is there such a link? I think you might need to pay for this.

Reply:
If the link costs, then never mind. That said, part of my asking for a link was to ask for a little more detail to help the reader know which ANSI SQL spec (or standard) you are referring them to. Which version is relevant for Presto?  [SQL:2023](https://en.wikipedia.org/wiki/SQL:2023), or one of the other eight versions of the standard?

Reply:
It's worth specifying that. Do we make any promises about whgich version of ANSI SQL Presto supports?

Reply:
I don't know which version of ANSI SQL Presto supports. If we don't support a specific version, then perhaps "Functions defined by the ANSI SQL spec" does not help the reader know if a function they are working on should be written as a core function and this might need rephrasing or removal. 

@tdcmeehan, is there any guidance you can give as to which if any specific version of ANSI SQL standard Presto supports?

Reply:
Perhaps we can just generically link to this reference: https://jakewheat.github.io/sql-overview/

Reply:
Okay, that works for me given the context! Added comment with suggested change below.